README.md

```diff
@@ -16,11 +16,7 @@ A few things you might want to know about `jiant`:
 
 To run a simple example demo experiment using data from GLUE:
```

Comment:
Emphasize that this is where the main setup instructions live.

Reply:
It seems very odd to leave this as multiple lines/sentences, but we can clean this up later if needed.